src/main/java/no/difi/asic/AsicUtils.java

```diff
@@ -120,21 +124,20 @@ public static void combine(OutputStream outputStream, InputStream... inputStream
     }
 
     public static MimeType detectMime(String filename) throws IOException {
```

Comment:
Det er ganske mange som bruker dette biblioteket i ulike sammenhenger, som risikerer å få anna content type med anna implementasjon, med ukjent konsekvens. Det vil være ei tryggare endring å fortsatt benytte `Files.probeContentType` for så å falle tilbake på Tika dersom førstnevnte ikkje gir resultat.